assets/svgbob_worker.js

```diff
@@ -0,0 +1,150 @@
+let importObject = {
```

Comment:
Found plenty of `let`s here and there. Would be nice to use `const` if not mutating so I don't have think that it's going to be modified.

Reply:
I was not 100% sure of const is going to be allowed in non transpiled code. 

I guess it is pretty green, https://caniuse.com/const will change a few around